opensrs/opensrsapi.py

```diff
@@ -405,6 +411,8 @@ def register_domain(self, domain, purchase_period, user, user_id,
                             'Invalid syntax on domain')):
                     raise errors.InvalidDomain(e)
                 raise errors.DomainRegistrationFailure(e)
+            if e.response_code == self.CODE_CLIENT_TIMED_OUT:
+                raise errors.DomainRegistrationTimedOut(e)
```

Comment:
Do you think it will respond with proper `response_code` for the timeout case? I'm expecting something like https://sentry.io/yola/domainservice/issues/117696697/